control-plane/common-outputs.tf

```diff
@@ -0,0 +1,73 @@
+output "k8s_distribtion" {
+  value = var.k8s_distribution
+}
+
+output "rancher_admin_password" {
+  value     = var.rancher_password
+  sensitive = true
+}
+
+output "rancher_url" {
+  value = local.rancher_url
+}
+
+output "rancher_token" {
+  value     = nonsensitive(local.rancher_token)
+  sensitive = false
+}
+
+output "install_rancher" {
+  value = var.install_rancher
+}
+
+output "install_certmanager" {
+  value = var.install_certmanager
+}
+
+output "install_monitoring" {
+  value = var.install_monitoring
+}
+
+output "certmanager_version" {
+  value = var.certmanager_version
+}
+
+output "tls_cert_file" {
+  value     = var.tls_cert_file
+  sensitive = true
+}
+
+output "tls_key_file" {
+  value     = var.tls_key_file
+  sensitive = true
+}
+
+output "rancher_charts_repo" {
+  value = var.install_monitoring ? var.rancher_charts_repo : null
+}
+
+output "rancher_charts_branch" {
+  value = var.install_monitoring ? var.rancher_charts_branch : null
+}
+
+output "rancher_version" {
+  value = var.rancher_version
+}
+
+output "use_new_bootstrap" {
+  value = local.use_new_bootstrap
+}
+
+output "kube_config_path" {
+  value = abspath(module.generate_kube_config.kubeconfig_path)
+}
+
```

Comment:
Nit: Let's get rid of miscellaneous extra white space

Reply:
We should add some kind of linting to fix these, possibly through pre-commit.

Reply:
I actually do have this setup but it has more things thrown in than just linting so I'll need to make a task to fix it up and add it to the repo